airbyte-ci/connectors/pipelines/pipelines/cli/dagger_run.py

```diff
@@ -89,17 +93,21 @@ def check_dagger_cli_install() -> str:
     return dagger_path
 
 
-def main():
-    set_working_directory_to_root()
+def mark_dagger_wrap():
+    """
+    Mark that the dagger wrap has been applied.
+    """
+    os.environ[DAGGER_WRAP_ENV_VAR_NAME] = "true"
+
+
+def call_current_command_with_dagger_run():
+    mark_dagger_wrap()
     if (os.environ.get("AIRBYTE_ROLE") == "airbyter") or (os.environ.get("CI") == "True"):
         os.environ[DAGGER_CLOUD_TOKEN_ENV_VAR_NAME_VALUE[0]] = DAGGER_CLOUD_TOKEN_ENV_VAR_NAME_VALUE[1]
 
     exit_code = 0
-    if len(sys.argv) > 1 and any([arg in ARGS_DISABLING_TUI for arg in sys.argv]):
-        command = ["airbyte-ci-internal"] + [arg for arg in sys.argv[1:] if arg != "--no-tui"]
-    else:
-        dagger_path = check_dagger_cli_install()
-        command = [dagger_path, "run", "airbyte-ci-internal"] + sys.argv[1:]
+    dagger_path = check_dagger_cli_install()
+    command = [dagger_path, "run"] + sys.argv
```

Comment:
nit: I think this change breaks the feature of running `airbyte-ci` from nested airbyte subfolders **if the binary is not in the shell path**:
e.g running: 
```bash
cd airbyte-ci/connectors/pipelines
./dist/airbyte-ci connectors --name=source-faker test
```

Output:
``` bash
 INFO     pipelines.cli.dagger_run: Running command: ['/Users/augustin/bin/dagger', 'run', './dist/airbyte-ci', 'connectors', '--name=source-faker', 'test']                       dagger_run.py:113
█ [0.00s] ERROR ./dist/airbyte-ci connectors --name=source-faker test
```

This is not happening after I added `airbyte-ci` to my path (which the local install process should do)